job_application/tests.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import datetime
+from jobboard.models import Job
+from django.db.models import QuerySet
+from django.contrib.auth.models import User
+from job_application.models import Application
+from student.models import Student, EducationalInstitution
+
+
+# The tests use the data that is already in the db from the migration files
+@pytest.fixture
+def test_institution():
+    test_institution = EducationalInstitution(name="test_institution")
+    test_institution.save()
+    return test_institution
+
+
+@pytest.fixture
+def test_student(test_institution):
```

Comment:
Maybe change the fixture name to example_student and example_job_application (As suggested by the mentors on previous PRs).

Reply:
done that :)